app/test_main.py

```diff
@@ -1,3 +1,24 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,result",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2])
+    ]
+)
```

Comment:
For better readability, you need to add an "id" with a description of the check. Example:
pytest.param(0, 0, [0, 0], id="both ages zero") or add ids=[<test description 1>, <test description 2>.......]